microdnf/main.cpp

```diff
@@ -101,6 +101,22 @@ static bool parse_args(Context & ctx, int argc, char * argv[]) {
         return true;});
     microdnf->register_named_arg(setopt);
 
+    auto & config = ctx.base.get_config();
+
+    auto assume_yes = ctx.arg_parser.add_new_named_arg("assumeyes");
+    assume_yes->set_long_name("assumeyes");
+    assume_yes->set_short_name('y');
+    assume_yes->set_short_description("automatically answer yes for all questions");
+    assume_yes->set_const_value("true");
+    assume_yes->link_value(&config.assumeyes());
```

Comment:
Is it used for autoupdate of conf from value from arg_parser?

Reply:
If the argument "-y"/"--assumeyes" is used, than value `true` (defined by `set_const_value`) will be stored into configuration option `assumeyes`.

Reply:
So, yes.